_projects/2024/100542532/100542532.Rmd

```diff
@@ -0,0 +1,532 @@
+---
+title: "Exploring Where Americans Can Afford to Live Alone"
+description: | 
+  Rising rents and stagnant wages make even modest apartments unaffordable in many U.S. cities
+categories: "2024"
+author: Isabel Monge
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+This double lollipop graph below, [originally published by The Economist in 2023](https://www.economist.com/graphic-detail/2023/08/04/our-carrie-bradshaw-index-where-americans-can-afford-to-live-solo), compares the median wage to the affordable studio wage across cities in the United States. It highlights the growing housing affordability crisis and reveals where single renters face the greatest financial challenges. My objective is to recreate this graph as closely as possible using R, and then to propose an alternative visualization.
+
+```{r, echo=FALSE, out.width='100%'}
```

Comment:
Please add the options `out.extra="class=external"` and `fig.cap="some text here to show as the caption of the image"`.